onedal/common/_policy.py

```diff
@@ -31,7 +31,10 @@ def _get_policy(queue, *data):
                 return _HostInteropPolicy()
             return _DataParallelInteropPolicy(data_queue)
         return _DataParallelInteropPolicy(queue)
-    assert data_queue is None and queue is None
+    if not (data_queue is None and queue is None):
+        raise RuntimeError(
+            "Operation using the requested SYCL queue requires the DPC backend"
+        )
```

Comment:
I think it worth to add else here. This not abvous that this is related with dpc_backend
```suggestion
    else:
        if not (data_queue is None and queue is None):
            raise RuntimeError(
                "Operation using the requested SYCL queue requires the DPC backend"
            )
```